tools/deployment/package-helm/templates/webui-deployment.yaml

```
@@ -17,6 +17,10 @@ spec:
         {{- include "clp.labels" . | nindent 8 }}
         app.kubernetes.io/component: "webui"
     spec:
+      {{- include "clp.createSchedulingConfigs" (dict
+          "root" .
+          "component" "webui"
+        ) | nindent 6 }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)